src/main/resources/alfresco/ibatis/org.alfresco.repo.domain.dialect.Dialect/node-delete-SqlMap.xml

```diff
@@ -9,9 +9,9 @@
         delete from alf_node
         where
             type_qname_id = #{typeQNameId} and
-            transaction_id <=
+            transaction_id IN
```

Comment:
This IN clause could generate a very big list as opposed to being an index scan.